src/main/java/KieTwoForOne.java

```diff
@@ -0,0 +1,171 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class KieTwoForOne {
+
+    private static ArrayList<Task> tasks = new ArrayList<>(100);
```

Comment:
The number 100 is a magic number. 
Consider defining it as a constant with a meaningful name, such as MAX_TASKS?